### PR TITLE
Add logging to CyberSource submit view

### PR DESCRIPTION
**Pull request by rlucioni**
These log messages are meant to help verify that this view is being hit consistently by users' browsers.

LEARNER-1671

### CODE REVIEW

File: ecommerce/extensions/payment/views/cybersource.py [hunk 1]
```diff
@@ -83,6 +94,11 @@ def form_invalid(self, form):
         return JsonResponse({'field_errors': errors}, status=400)
 
     def form_valid(self, form):
+        logger.info(
+            'Valid payment form submitted for basket [%d].',
+            basket.id
```
Comment (mjfrey) [inline ecommerce/extensions/payment/views/cybersource.py]: Maybe  move this down to after generating the params and log the transaction ID?

Comment (rlucioni) [reply ecommerce/extensions/payment/views/cybersource.py]: @mjfrey moving, I also missed that the basket variable isn't yet available here.

File: ecommerce/extensions/payment/views/cybersource.py [hunk 2]
```diff
@@ -73,6 +79,11 @@ def _basket_error_response(self, error_msg):
         return JsonResponse(data, status=400)
 
     def form_invalid(self, form):
+        logger.info(
+            'Invalid payment form submitted for basket [%d].',
+            basket.id
```
Comment (schenedx) [inline ecommerce/extensions/payment/views/cybersource.py]: Where does this `basket` variable come from?

File: ecommerce/extensions/payment/views/cybersource.py [hunk 3]
```diff
@@ -59,6 +59,12 @@ class CybersourceSubmitView(FormView):
 
     @method_decorator(login_required)
     def dispatch(self, request, *args, **kwargs):
+        logger.info(
+            'CyberSource submit view called for basket [%d]. It is in the [%s] state.',
+            basket.id,
```
Comment (schenedx) [inline ecommerce/extensions/payment/views/cybersource.py]: Where does the variable `basket` come from?

**Review by schenedx — COMMENTED** [review]
I got a few questions

File: ecommerce/extensions/payment/views/cybersource.py (hunk 3 above)
Comment (rlucioni) [reply ecommerce/extensions/payment/views/cybersource.py]: @schenedx it doesn't, see above. This is what happens when you try to do too many things at once 😟 . Fixing.

**Comment by rlucioni** [comment]
@mjfrey @schenedx I've addressed your comments.

**Review by mjfrey — APPROVED** [review]

**Review by schenedx — APPROVED** [review]
👍

**Review by clintonb — APPROVED** [review]

**Comment by rlucioni** [comment]
I added one additional message here to the checkout view, just to help rule it out.

**Comment by rlucioni** [comment]
Looks like Codecov is having problems: https://status.codecov.io/.

**Pull request merged**